mobile_back_qti/make/qti_backend.mk

```diff
@@ -50,11 +50,28 @@ else ifeq ($(WITH_QTI),$(filter $(WITH_QTI),1 2))
   endif
 
   backend_qti_windows_files=${BAZEL_LINKS_PREFIX}bin/mobile_back_qti/cpp/backend_qti/libqtibackend.dll \
-    ${local_snpe_sdk_root}/lib/aarch64-windows-vc19/SNPE.dll \
-    ${local_snpe_sdk_root}/lib/aarch64-windows-vc19/SnpeHtpV68Stub.dll \
-    ${local_snpe_sdk_root}/lib/dsp/libSnpeHtpV68Skel.so \
+    ${local_snpe_sdk_root}/lib/aarch64-windows-msvc/SNPE.dll \
+    ${local_snpe_sdk_root}/lib/aarch64-windows-msvc/SnpeHtpV68Stub.dll \
+    ${local_snpe_sdk_root}/lib/aarch64-windows-msvc/SnpeHtpPrepare.dll \
+    ${local_snpe_sdk_root}/lib/hexagon-v68/unsigned/libSnpeHtpV68Skel.so \
     mobile_back_qti/run_performance_tests.bat \
     mobile_back_qti/run_accuracy_tests.bat
   backend_qti_windows_target=//mobile_back_qti/cpp/backend_qti:libqtibackend.dll
   backend_qti_filename=libqtibackend
 endif
+
+.PHONY: flutter/libqtibackend/patchlibs
+flutter/libqtibackend/patchlibs:
```

Comment:
@freedomtan: Should this be a permanent solution and to be merged into the `master` branch?

Reply:
IMHO, we can merge this for now, and remove this once we move to the newer ndk.